TACC/Core-CMS
# feat: poetry 1.4.0

## Overview / Changes

Update Poetry to 1.4.0.

## Related

- mimics poetry update in #626 (used on tup)
- mimics poetry update in #625
- similar to https://github.com/TACC/TACC-Docs/commit/2c4907f
- similar to https://github.com/DesignSafe-CI/DS-User-Guide/pull/2

## Testing

1. Build a local CMS
2. Verify build completes.
3. Verify setup is unhindered.
4. Verify site renders.

## UI

![core-cms-653](https://github.com/TACC/Core-CMS/assets/62723358/14cd1302-cd33-42fd-a67c-8c1fa169e1df)
